### PR TITLE
UI widgets: remove unnecessary setLayout

### PR DESCRIPTION
**Pull request by deanlee**

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
We do this in all the widgets right now, although unnecessary because we set the layout parent. Any docs on the best practice here? If we decide to go with this, let's fix all instances.

**Comment by deanlee** [comment]
I think construct it with parent,and unified name to main_layout will be clearer.

**Comment by adeebshihadeh** [comment]
I'm fine with that. Let's make it consistent across all widgets.

File: selfdrive/ui/qt/maps/map.cc [hunk 1]
```diff
@@ -441,18 +441,19 @@ void MapWindow::offroadTransition(bool offroad) {
...
     icon_01 = new QLabel;
     layout->addWidget(icon_01);
     layout->addStretch();
-    layout_outer->addLayout(layout);
+    main_layout->addLayout(layout);
   }
 
   {
-    QVBoxLayout *layout = new QVBoxLayout;
+    QWidget * w = new QWidget;
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/maps/map.cc:455]:
```suggestion
    QWidget *w = new QWidget;
```

**Pull request merged**